src/css/modules/_TileGroup.scss

```diff
@@ -16,3 +12,28 @@
     }
   }
 }
+
+.coa-TileGroup--border {
+  border: 4px solid $coa-color-blue;
+  border-radius: 6px;
```

Comment:
super picky -- can we make these more tile group variables. so if we have border-radiuses elsewhere we can keep consistent.